platform/windows/detect.py

```diff
@@ -294,7 +294,10 @@ def configure_mingw(env):
     ## Compiler configuration
 
     if (os.name == "nt"):
-        env['ENV']['TMP'] = os.environ['TMP']  # way to go scons, you can be so stupid sometimes
```

Comment:
This was redundant with a call already made in `configure()` (and we can do without the comment...).